File: AllspiceTheSecond/Models/Recipe.cs
namespace AllspiceTheSecond.Models
{
  public class Recipe
    {
        public int id { get; set; }
        public string title { get; set; }
        public string description { get; set; }
        public string img { get; set; }
        public string category { get; set; }
        public string creatorId { get; set; }
        public Account creator { get; set; }
    }
}

File: AllspiceTheSecond.client/src/models/Recipe.js

export class Recipe{

  constructor(data){
    this.id = data.id
    this.title = data.title
    this.img = data.img
    this.category = data.category
    this.creatorId = data.creatorId
    this.creator = data.creator || null
    this.favoriteId = data.favoriteId || null
  }

}
